style.css

```diff
@@ -0,0 +1,201 @@
+  .wrap {
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/VeronikasBaliProject/pullRequest?prid=5927042)